### ISSUE TITLE
Passing in initialCroppedAreaPixes calls onCropChange

### ISSUE THREAD

**Issue by Alba-C** [ricardo-ch/react-easy-crop#39]
## Passing in initialCroppedAreaPixes calls onCropChange

Thanks you so much for this tool.  I'm running into an issue when allowing a user to edit an already cropped image.  We're passing in the initialCroppedAreaPixels and that is working fine to load the current image.  However, both onCropChange and  onCropComplete are being called when this happens.  

I need to be able to determine actual user interactions from the component being set up with initialCroppedAreaPixels.  I was expecting onCropComplete to only run when the user stops zooming or dragging, but it's getting called during the initial setup too.  

User Journey:
-User crops an image and is then prompted to save cropped image.
-User opens an already cropped image to continue editing.
 > expected behavior: user is only prompted to save again, if the user has made a change.
 > actual behavior: user is prompted to save without have made any changes, because no way to distinguish between user interaction and initial setup.

**Comment by ValentinH** [comment]
Hello, thanks for reporting this issue. Indeed, I don't think current behavior is correct. I'll look into it.

**Comment by ValentinH** [comment]
After digging into it, the `onCropComplete` callback can actually even be called several times during initialisation. it's actually used to the parent container to always know the current crop rectangle.

I think the best for you would be to create a dedicated callback like `onInteractionEnd()`; Would it make sense for you?

**Comment by Alba-C** [comment]
Yes, if onInteractionEnd() is specifically user generated actions that's great.  As long as we can know when user drag and zoom actions end, then we can prompt the user to save their changes.

**Comment by ValentinH** [comment]
Would you like to submit a PR for this?

**Comment by Alba-C** [comment]
I don't know if I'll have time in the short term.

**Comment by ValentinH** [comment]
Ok, it depends on how quickly you need this. I'll probably work on this this weekend...

**Comment by Alba-C** [comment]
I just submitted a  [PR for this.](https://github.com/ricardo-ch/react-easy-crop/pull/42)

I was thinking that our main objective can be accomplished as long as we know when a user interaction began instead of when it ended (which would probably be more useful in the long term).  I tested it locally and triggers on drag, pinch, zoom and wheel.   

Take a look and let me know if you think this will work.
